app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,58 @@
+class FriendshipsController < ApplicationController
+  def new
+    @users = User.all_except(current_user)
+  end
+
+  def create
+    user = current_user
+      @friend = Friendship.create(user_id: user.id, friend_id: params[:friend_id])
```

Comment:
- This variable should be named as `@friendship` as its an object of `Friendship` class
- you're first creating and then trying to save which is of no use, for that purpose use `.new`/`.build` methods to initialize an object and use `.save` with them

So this can be changed to something like:
```ruby
@friendship = current_user.friendships.new(friend_id: params[:friend_id])
```